src/Akka.Streams.Kafka/Stages/CommitableConsumerStage.cs

```diff
@@ -0,0 +1,179 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Akka.Streams.Kafka.Messages;
+using Akka.Streams.Kafka.Settings;
+using Akka.Streams.Stage;
+using Confluent.Kafka;
+
+namespace Akka.Streams.Kafka.Stages
+{
+    internal class CommitableConsumerStage<K, V, Msg> : GraphStageWithMaterializedValue<SourceShape<Msg>, Task>
```

Comment:
This class is similar to `PlainSource`, but I don;t want to merge them until API will be finished

Reply:
Why do your stages return a finished Task as materialized value?

Reply:
It is a temporary solution. I should fix it